pkg/alerter/alerter.go

```diff
@@ -0,0 +1,79 @@
+package alerter
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/cloudskiff/driftctl/pkg/resource"
+	"github.com/sirupsen/logrus"
+)
+
+type Alerter struct {
+	sync.Mutex
+	Alerts   Alerts
+	AlertsCh chan Alerts
+}
+
+func NewAlerter() *Alerter {
+	return &Alerter{
+		Alerts:   make(Alerts),
+		AlertsCh: make(chan Alerts),
+	}
+}
+
+func (a *Alerter) Start() {
+	go func() {
+		defer close(a.AlertsCh)
+		for {
+			alert := <-a.AlertsCh
+			a.mergeAlerts(alert)
+		}
+	}()
+}
+
+func (a *Alerter) Alert(key string, alert Alert) {
+	a.AlertsCh <- Alerts{
```

Comment:
Why not doing a `AlertsCh chan Alert` instead of `AlertsCh chan Alerts`

Reply:
I did an `AlertsCh chan Alerts` so that I don't need to send into one function channel or two differents channels and the key and the alert (this is just a message and a bool) to create the map of alerts.